### DISCUSSION TITLE
Problem with scene loading

### DISCUSSION THREAD

**Discussion by Rallmir** [michalzc/oq-system#116] [Ideas]
Well done and thank you very much for your work on OpenQuest.
I'm having a problem with scene loading that doesn't load completely but stays stuck at 98%.
I've never had this problem on other Foundry's systems.
But i always have it on OpenQuest, despite creating several worlds.
I don't know if it's something to do with the beta.
Thanks in advance.

**Comment by michalzc** [comment]
Do you have any modules enabled?
Could you pack your world and send it to me?

**Comment by Rallmir** [comment]
Thanks for your quick reply.I have deleted the worlds that had the scene loading problem (I had made 3 successive worlds that had the same problem).At first, it seemed to work, then when I reloaded the world, the scene no longer loaded.I've just created a new one and I'll keep you posted (and send you the world).
In terms of active modules, I have :- autocomplete whisper- babele- chat portrait- dice so nice-dice tray- FR traduction du core- libWrapper- Ownership viewer-popout!

**Comment by Rallmir** [comment]
Hey! I may have found it.When I reloaded the world, I placed a character's token on the scene.(nb : You can't see it if it's the default image)But once the character's token image is changed, the token appears on scene.However, when I reload the world, I get the 98% loading bug!)Which may explains why there's no bug when creating scenes, but once you try to place a token... (in my opinion, but I'm technically rubbish)^^I hope this helps.

**Comment by michalzc** [comment]
Thank you for the info. I will try to replicate the problem.

**Comment by michalzc** [comment]
I've loaded the same modules and on my side, all scenes load properly. Could you check the developer console? Hit F12 on Chrome or related browsers, click the right mouse button somewhere on the console tab, choose 'save as...' and provide the saved file.

**Comment by Rallmir** [comment]
Hi, I've just done the test.
The problem is with the browser. 
I was using Firefox. I still have the display problem linked (I think) to the character token.
I immediately launched the game with Chrome : no problem.
